src/classes/Batchs/DeleteLogsBatch.cls

```diff
@@ -0,0 +1,34 @@
+/**
+ * Created by Mariya on 19.01.2023.
+ */
+
+public with sharing class DeleteLogsBatch implements Database.Batchable<sObject>, Schedulable, Database.Stateful {
+
+    private Logger logger;
+
+    public Iterable<sObject> start(Database.BatchableContext param1) {
+        return Database.getQueryLocator('SELECT CreatedDate FROM Log__c WHERE CreatedDate < N_DAYS_AGO:7');
```

Comment:
MINOR
Please make number of days as property that can be passed via job initialization through the class constructor.
Also, use new feature:
https://help.salesforce.com/s/articleView?id=release-notes.rn_apex_bind_var_soql.htm&type=5&release=242